src/pages/settings/InitialSettingsPage.js

```diff
@@ -78,39 +87,6 @@ const defaultProps = {
     ...withCurrentUserPersonalDetailsDefaultProps,
 };
 
-const defaultMenuItems = [
```

Comment:
Why was this moved?

Reply:
The `brickRoadIndicator: PaymentMethods.hasPaymentMethodError(props.bankAccountList, props.cardList) ? 'error' : null,` needs access to something in `props` now